Run `update` first in `make package` cmd.

simitt opened the pull request:
Avoid failure based on missing files or setup that are be added by running `make
update`.
Closes https://github.com/elastic/apm-server/issues/355.

ruflin commented on libbeat/scripts/Makefile:
```diff
@@ -457,7 +457,7 @@ package-setup:
 
 .PHONY: package
 package: ## @packaging Create binary packages for the beat.
-package: package-setup
```
I wonder if we should even run `clean` before `update` to make sure everything is "fresh".

simitt commented:
@ruflin I rebased to master and added a `clean` before the `update` cmd. Running `make package` did not lead to any errors when running in the beats root folder.

ruflin commented:
@simitt Thanks. Lets keep an eye on the jenkins package jobs.
